### ISSUE TITLE
babel-loader version clashing with @storybook/react's version

### ISSUE THREAD

**Issue by jonathanadler** [wix/yoshi#971]
# 🐛 Bug Report

### 😯 Current Behavior

Using the latest @storybook version (`4.1.11`) and latest Yoshi (`3.30.0`) results with an error:
![image](https://user-images.githubusercontent.com/3277080/52345255-64836500-2a25-11e9-8b95-0dd8e1050804.png)
Due to clashing versions of `babel-loader`.
You can see more information about this here:
https://github.com/storybooks/storybook/issues/5183

### 🤔 Expected Behavior
Should run and open storybook

### 💁 Possible Solution
Using older versions of Storybook and Yoshi

### 👣 Steps to Reproduce
Create new client project.
Install `storybook/react`
Add a `.storybook` directory with a `config.js` file, such as:
```
import { configure } from '@storybook/react';

function loadStories() {
  require('../stories/index.js');
}

configure(loadStories, module);
```
and a `webpack.config.js` file like this:
```
const yoshiWebpackConfig = require('yoshi/config/webpack.config.storybook');

module.exports = (config, env, defaultConfig) => {
  return yoshiWebpackConfig(defaultConfig);
};
```
Run `start-storybook -p 9001 -c .storybook`

### 💻 Code Sample
Will be given by demand

### 🌍 Your Environment
```
 System:
    OS: macOS 10.14.3
    CPU: (8) x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
  Binaries:
    Node: 10.14.2 - ~/.nvm/versions/node/v10.14.2/bin/node
    Yarn: Not Found
    npm: 6.4.1 - ~/.nvm/versions/node/v10.14.2/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Browsers:
    Chrome: Not Found
    Firefox: 63.0.3
    Safari: 12.0.3
  npmPackages:
    @storybook/core:  4.0.6 
    @storybook/react: 4.0.6 => 4.0.6 
    babel-preset-yoshi:  3.17.0 
    eslint-config-yoshi:  3.25.1 
    eslint-config-yoshi-base:  3.25.1 
    jest-environment-yoshi-bootstrap:  3.30.0 
    jest-environment-yoshi-puppeteer:  3.30.0 
    jest-yoshi-preset: ^3.5.0 => 3.30.0 
    storybook: Not Found
    tslint-config-yoshi:  3.26.0 
    tslint-config-yoshi-base:  3.26.0 
    typescript: Not Found
    webpack:  4.28.4 
    yoshi-config:  3.30.0 
    yoshi-helpers:  3.30.0 
    yoshi-runtime: ^1.0.41 => 1.0.42 

 Yoshi config
{
  "entry": {
    "app": "./client",
    "mySites": "./mySites"
  },
  "experimentalServerBundle": true,
  "externals": {
    "react": "React",
    "react-dom": "ReactDOM"
  },
  "hmr": "auto"
}
```

**Comment by ronami** [comment]
I think this was solved in #1095 (v4.1.0-alpha.6), can you please check?

**Comment by jonathanadler** [comment]
@ronami yes it was! Thanks!
